zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -451,46 +491,44 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
         };
 
         self.tx_id
-            .assign(region, offset, Value::known(F::from(tx.id as u64)))?;
+            .assign(region, offset, Some(tx.id.to_le_bytes()))?;
```

Comment:
I think there are quite a few of these assign calls of the `RandomLinearCombination` that take an array of bytes, but I think many times we have integers instead of byte arrays.  Maybe we can add a method like `assign_h160`, `assign_usize` to `RandomLinearCombination` to avoid calling `foo.to_le_bytes()` all the time.

Reply:
Added intDecomposition type https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1411/commits/87370f81b0f15c6809fe0a14e908b6c0aa2cdb78